utils/staking-miner/src/main.rs

```diff
@@ -586,7 +629,7 @@ async fn main() {
 	log::info!(target: LOG_TARGET, "connected to chain {:?}", chain);
 
 	any_runtime_unit! {
-		check_versions::<Runtime>(&rpc).await
+		check_versions::<Runtime>(&rpc).await.expect("Runtime version is invalid, terminating")
```

Comment:
no -- we just spoke of this, we only need to check this strictly in `monitor` in other commands it can be a mismatch.